File: lab_2/output/input1.py
@dataclass
class User:
    id: float
    name: str
    email: str
    age: float
    isActive: bool

@dataclass
class Product:
    id: float
    title: str
    price: float

def createUser1(id: float, name: str, email: str, age: float) -> User:
    return {
    id: id,
    name: name,
    email: email,
    age: age,
    isActive: True
    }

def calculateDiscount1(price: float, percent: float) -> float:
    return price - price * percent / 100

isAdult1 = lambda age: age >= 18  # type: (float) -> bool

increasePrice1 = lambda price: price + 10  # type: (float) -> float

def printUserName1(user: User) -> None:
    print(user.name)

File: lab_2/input/input1.ts
interface User {
  id: number;
  name: string;
  email: string;
  age: number;
  isActive: boolean;
}

interface Product {
  id: number;
  title: string;
  price: number;
}

function createUser1(
  id: number,
  name: string,
  email: string,
  age: number
): User {
  return {
      id: id,
      name: name,
      email: email,
      age: age,
      isActive: true
  };
}

function calculateDiscount1(
  price: number,
  percent: number
): number {
  return price - price * percent / 100;
}

const isAdult1 = (age: number): boolean => age >= 18;

const increasePrice1 = (price: number): number => price + 10;

const printUserName1 = (user: User): void => {
  console.log(user.name);
};
